extensions/smallrye-fault-tolerance/deployment/src/main/java/io/quarkus/smallrye/faulttolerance/deployment/NoopMetricRegistry.java

```diff
@@ -0,0 +1,231 @@
+package io.quarkus.smallrye.faulttolerance.deployment;
+
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.SortedSet;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import org.eclipse.microprofile.metrics.*;
+
+import io.quarkus.runtime.StartupEvent;
+
+@ApplicationScoped
+public class NoopMetricRegistry extends MetricRegistry {
+
+    void onStart(@Observes StartupEvent ev) {
+        //the following sys prop will de-activate fault tolerance metrics collector
+        System.setProperty("MP_Fault_Tolerance_Metrics_Enabled", "false");
```

Comment:
I don't know anything about Quarkus, but I'd expect this configuration change to be done much earlier (during build).

Reply:
Build is done on a separate JVM (except on dev mode), so it's not possible. But maybe this can be checked by someone with good knowledge on Quarkus configuration because I don't like neither to use a StartupEvent to set a sys prop but I didn't find an easier/more elegant way to do this.

Reply:
Can't we use a `io.quarkus.deployment.builditem.SystemPropertyBuildItem` to do this?

Reply:
@jmartisk of course yes :)
I search for such build item yesterday and didn't find any ... I missed it I don't know how ...

Reply:
So glad I have IntelliJ IDEA with its super sophisticated class searching mechanism where I could write "propertybuild" and received the answer :)

Reply:
> So glad I have IntelliJ IDEA with its super sophisticated class searching mechanism where I could write "propertybuild" and received the answer :)

I too use IDEA ... maybe this is not my IDE fault but ... mine ;)